projects/tomofinance.js

```diff
@@ -0,0 +1,23 @@
+const { getBlock } = require('./helper/getBlock')
+const sdk = require('@defillama/sdk');
+
+async function taiSupply(timestamp, ethBlock, chainBlocks) {
+    const block = await getBlock(timestamp, 'tomochain', chainBlocks);
+    const taiContract = '0x4d04315e6BABD038d15ea0E240A88B292d2Add41';
+
+    const supply = (await sdk.api.erc20.totalSupply({
```

Comment:
since this seems to be a CDP-based stable, shouldn't we count the coins that are collateral instead of the stablecoins issued?

Reply:
I can't find the deposits anywhere, I've DMd the team to ask so hopefully they'll respond